.azure/azure-pipelines-mac.yml

```diff
@@ -42,14 +51,15 @@ jobs:
   steps:
     - script: |
         set -ex
-        PACKAGES="glib gobject-introspection libsndfile pkg-config jack dbus-glib pulseaudio portaudio sdl2 libomp"
+        PACKAGES="glib gobject-introspection libsndfile pkg-config jack dbus-glib pulseaudio portaudio sdl2 libomp readline"
         brew install $PACKAGES
       displayName: 'Prerequisites'
     - script: |
         set -ex
         mkdir build && cd build
-        export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig"
-        cmake -Werror=dev $(CMakeFlags) -DCMAKE_BUILD_TYPE=Release -DCMAKE_VERBOSE_MAKEFILE=1 -DNO_GUI=1 ..
+        export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig:/usr/local/opt/readline/lib/pkgconfig"
+        # assist cmake in finding openmp installed via homebrew
+        cmake -Werror=dev $(CMakeFlags) -DCMAKE_BUILD_TYPE=Release -DCMAKE_VERBOSE_MAKEFILE=1 -DNO_GUI=1 -DCMAKE_PREFIX_PATH=/usr/local/opt/libomp ..
```

Comment:
It may be preferable to use `$(brew --prefix libomp)` instead of hardcoding the path. It should make it a little more future-proof if either Homebrew or Azure changes their layout.

Reply:
Done, as well as for readline and libffi. Thanks!